sdk-lib/src/main/java/cash/z/ecc/android/sdk/block/processor/CompactBlockProcessor.kt

```diff
@@ -1196,145 +1856,84 @@ class CompactBlockProcessor internal constructor(
     /**
      * Rewind back at least two weeks worth of blocks.
```

Comment:
`rewindToNearestHeight` rewinds to the "nearest" supported height. Is that guaranteed to be _at least_ two weeks back?

Reply:
It depends on the Rust implementation of `getNearestRewindHeight.` Kotlin side guarantees that (except the activation height limit case). So, I slightly changed the comment to reflect this. Thanks!